scripts/combine_labels.py

```diff
@@ -101,6 +106,12 @@ def main(args):
                       labelers for a record to be considered anomalous (should \
                       be a number between 0 and 1)")
   
+  parser.add_argument("--windowSize",
+                      default=0.10,
+                      type=float,
+                      help="Set the size of the standard anomaly window, as a \
```

Comment:
Okay, this will be removed following your previous comment -- to delete the windowSize user parameter.